am/templates/api/api-configmap.yaml

```diff
@@ -12,25 +12,50 @@ metadata:
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
 data:
   gravitee.yml: |
-    {{- if .Values.api.ssl.enabled }}
+    {{- if .Values.api.jetty }}
```

Comment:
I'm wondering how this change will work on an existing installation ?
Does it mean that users using `api.ssl.enabled: true` must switch to `api.jetty.secured: true` ?

Reply:
In this case yes.

Reply:
Agree with @jhaeyaert , we should keep consistency with all others products. 

I'm still not sure if we have to use : `Values.api.jetty.*` rather than `Values.api.http.*` 🤔 

@jhaeyaert what do you think ?

Reply:
Yeah I think keeping `Values.api.http.*` is better as we do not introduce a thigh coupling with the underlying server we are using for our api (ie: jetty). Also, it avoid introducing breaking changes so...

Reply:
@jhaeyaert @tcompiegne fair enough

I'll put the values under `Values.api.http.*` and I'll keep using the `ssl.enabled` flag